documentation/shared/attributes.adoc

```diff
@@ -42,14 +42,22 @@
 :kafka-exporter-project: link:https://github.com/danielqsj/kafka_exporter[Kafka Exporter^]
 
 //OAuth attributes and links
+:strimzi-project-name: Strimzi
+:keycloak-project-name: Keycloak
 :oauth2-site: link:https://oauth.net/2/[OAuth 2.0 site^]
 :oauth-version: 0.7.1
 :keycloak-server-doc: link:https://www.keycloak.org/documentation.html[Keycloak documentation^]
+:keycloak-server-install-doc: link:https://www.keycloak.org/docs/latest/server_installation/index.html#_operator[Installing the Keycloak Operator^]
 :keycloak-authorization-services: link:https://www.keycloak.org/docs/latest/authorization_services/index.html[Authorization Services^]
+:authz-services-model-ref: xref:con-oauth-authorization-services-model-{context}[Keycloak Authorization Services model for managing permissions]
```

Comment:
Why do we need a variable for an internal link? I think you should place it into the document directly.

Reply:
Because of 'Keycloak' in the link text. But you say that it's not necessary as it will be replaced by a preprocessing search/replace routine.

Reply:
In the past we decided to not use any variables for project names etc. It is unreliable and just makes writting the docs harder.